base/complex.jl

```diff
@@ -114,6 +114,7 @@ real(::Type{Complex{T}}) where {T<:Real} = T
     isreal(x) -> Bool
 
 Test whether `x` or all its elements are numerically equal to some real number.
```

Comment:
Perhaps this could be written shorter as:

```
Test whether `x` or all its elements are numerically equal to some real number, `Inf` or `NaN`.
```